.github/workflows/json.yaml

```diff
@@ -0,0 +1,25 @@
+name: JSON Validation
+
+on:
+  pull_request:
+    branches: [ "*" ]
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ "3.10" ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: JSON validation.
+        run: |
+          set -o pipefail
+          find . -name "*.json" -exec python -m json.tool {} 1>/dev/null \;
```

Comment:
Pytest unit test is more suitable. This way we can add more check on the KB. Like the common issue of the categories being broken.